octane/evmengine/keeper/helpers.go

```diff
@@ -41,3 +47,43 @@ func retryForever(ctx context.Context, fn func(ctx context.Context) (bool, error
 		return nil
 	}
 }
+
+// blobsBundleFromEth converts an engine.BlobsBundleV1 to a types.BlobsBundleV1.
+func blobsBundleFromEth(bundle *engine.BlobsBundleV1) *types.BlobsBundleV1 {
+	toByteSlices := func(in []hexutil.Bytes) [][]byte {
+		var out [][]byte
+		for _, i := range in {
+			out = append(out, i)
+		}
+
+		return out
+	}
+
+	return &types.BlobsBundleV1{
+		Commitments: toByteSlices(bundle.Commitments),
+		Proofs:      toByteSlices(bundle.Proofs),
+		Blobs:       toByteSlices(bundle.Blobs),
+	}
+}
+
+// blobHashes returns the hashes of the blobs if the blobs are valid.
+func blobHashes(bundle *types.BlobsBundleV1) ([]common.Hash, error) {
+	sidecar, err := bundle.EthSidecar() // Nil bundle is valid but empty.
+	if err != nil {
+		return nil, err
+	}
+
+	hashes := make([]common.Hash, 0, len(sidecar.Blobs))
+
+	hasher := sha256.New()
+	for i := range sidecar.Blobs {
+		if err := kzg4844.VerifyBlobProof(&sidecar.Blobs[i], sidecar.Commitments[i], sidecar.Proofs[i]); err != nil {
```

Comment:
Is verifying the whole blob, proof, and commitment required for all validators? Since only `Commitments` are required to calculate `versioned hashes` which are provided to geth via `NewPayload`, other fields (`Blobs and Proofs`) are technically unused. Can they not be removed?

Reply:
Did you find the answer to this?

Reply:
Yeah, we only include Commitments for minimal but safe support